src/Parser/Mage/Frost/Modules/Features/DeathTracker.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import CoreDeathTracker from 'Parser/Core/Modules/DeathTracker';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+
+class DeathTracker extends CoreDeathTracker {
+
+  suggestions(when) {
+    const timeDeadSeconds = this.timeDead / 1000;
+    const boss = this.owner.boss;
+    if (!boss || !boss.fight.disableDeathSuggestion) {
```

Comment:
Does the death event contain any info about how the player died? Is it possible to ignore specifically the "RP kill" but still detect the rest?

Reply:
The problem with Argus, is walking into the tree doesnt count as a "ressurection". So regardless, it will say you were dead from the RP to the end of the fight.

Reply:
Ohhh yeah that makes it difficult.